### ISSUE TITLE
Client crashes when opening Crystal Chest on 1.8 Server

### ISSUE THREAD

**Issue by MajorGeneralRelativity** [cpw/ironchest#75]
When opening a Crystal Chest on my server, i get a [Netty Client IO #7/Error]: There was a critical exception handling a packet on channel IronChestio. This can affect just person at a time, or multiple, and it tends to disconnect people multiple times. Let me know if there is anything i can do to fix this

**Comment by MajorGeneralRelativity** [comment]
Whoops, wrong place i think

**Comment by ThomasRules2000** [comment]
No, this is the right place - the github issue tracker for the iron chests mod
